tests/php/Control/DirectorTest.php

```diff
@@ -565,6 +565,7 @@ public function providerTestTestRequestCarriesGlobals()
      */
     public function testTestRequestCarriesGlobals($url, $fixture, $method)
     {
+        Controller::config()->set('add_trailing_slash', false);
```

Comment:
Why did this (and others in this file) get added?

Reply:
Unit tests were failing without it being added

Looking at this again, it would be clearer if I instead have `Director::config()->set('alternate_base_url', 'https://www.mysite.com');` (or whatever test website is used).  I may not work in all cases though I give it a go

Reply:
OK ... It's because I had local config setting it to true :1st_place_medal: